lib/modules/dosomething/dosomething_signup/includes/SignupsController.php

```diff
@@ -2,6 +2,8 @@
 
 class SignupsController extends EntityAPIController {
 
+  protected $ignore_global_user = FALSE;
```

Comment:
yeah fair, I am worried that this isn't really an experiment and we won't ever clean up and this introduces a lot of confusion/complexity to the code.